migrations/data-source.ts

```diff
@@ -0,0 +1,22 @@
+import { DataSource } from 'typeorm';
+
+function getEnvVariable(key: string): string {
+    const value = process.env[key];
+    if (value === undefined) {
+        throw new Error(`Environment variable ${key} not defined!`);
+    }
+    return value;
+}
+
+async function configureDataSource(): Promise<DataSource> {
+    return new DataSource({
+        database: getEnvVariable('DB_PATH'),
+        type: 'sqlite',
+        synchronize: false,
+        logging: false,
+        entities: ['src/**/**.entity.ts'],
```

Comment:
I don't think this is necessary

Reply:
I'm pretty sure that it's necessary, if we don't then how will Typeorm know which files to use for entity generation?